charts/node/values.yaml

```diff
@@ -316,176 +370,201 @@ node:
       #   vaultPath: kv/secrets/babeKey
       #   vaultKey: babe
 
-    ## Node key to use via vault
-    ##
+    # -- Node key to use via vault
     nodeKey: {}
       # name: nodekey
       # vaultPath: kv/secret/nodekey
       # Remark; vaultKey cannot contain dashes ('-'), only alphanumeric characters due to a limitation in the go templating
       # vaultKey: key
       # vaultKeyAppendPodIndex: false
 
-  ## If enabled, generate a persistent volume to use for the keys
-  ##
+  # -- If enabled, generate a persistent volume to use for the keys
   persistGeneratedNodeKey: false
-  # customNodeKey: ""
+  # -- Name of the secret containig the key
+  customNodeKey:
 
-  ## Expose metrics via Prometheus format in /metrics endpoint
-  ## Pass the following args:
-  ##   - "--prometheus-external" \
-  ##   - "--prometheus-port {{ .Values.node.prometheus.port }}"
-  ##
+  # -- Expose metrics via Prometheus format in /metrics endpoint.
+  # Passes the following args to the Polkadot binary:
+  #   - "--prometheus-external" \
+  #   - "--prometheus-port {{ .Values.node.prometheus.port }}"
   prometheus:
-    enabled: true
+    # -- Expose Prometheus metrics
+    enabled: false
+    # -- The port for exposed Prometheus metrics
     port: 9615
 
-  ## Enable Service Monitor of Prometheus-Operator
-  ## ref: https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/user-guides/getting-started.md#include-servicemonitors
-  ##
+  # -- Service Monitor of Prometheus-Operator
+  # ref: https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/user-guides/getting-started.md#include-servicemonitors
   serviceMonitor:
+    # -- Enables Service Monitor
     enabled: false
-    #namespace: monitoring
+    # -- Namespace to deploy Service Monitor. If not set deploys in the same namespace with the chart
+    namespace:
+    # -- Scrape interval
     interval: 30s
+    # -- Scrape timeout
     scrapeTimeout: 10s
+    # -- Labels to scrape
     targetLabels:
       - node
+    # -- Relabelings config
     relabelings: []
+    # -- Metric relabelings config
     metricRelabelings: []
 
+  # -- Annotations to add to the Service
   serviceAnnotations: {}
 
-  ## Use deprecated ws/rpc flags.
-  ## ref: https://github.com/paritytech/substrate/pull/13384
-  ##
+  # -- Use deprecated ws/rpc flags.
+  # ref: https://github.com/paritytech/substrate/pull/13384
   legacyRpcFlags: false
```

Comment:
This is good time to break rpc port backward comparability. We need to remove option `legacyRpcFlags`, cleanup all `httpPort` and `wsPort` and keep only `rpcPort`.

Reply:
Parity still runs some archive nodes on v0.9.42. Removing the flag will break the installations. I'd suggest to wait before v0.9.44 is released and we upgraded all the nodes to v0.9.43.